### PR TITLE
Added new question about agile methods

### PR DESCRIPTION
**Pull request by PhilipeSousa**
Added new question about agile methods, more specifically about Story Points.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -144,6 +144,17 @@ const agileQuiz = [
       "A user story is an informal, general explanation of a software feature written from the perspective of the end user.",
     Link: "https://www.freecodecamp.org/news/how-to-write-user-stories-epics-pesonas/",
   },
+  {
+    Question:
+      "Which of the following is NOT one of notable features of story points?",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:149]:
```suggestion
      "In Agile, which of the following is NOT one of notable features of story points?",
```

File: src/data/agile-quiz.ts [hunk 2]
```diff
@@ -144,6 +144,17 @@ const agileQuiz = [
...
+    Answer: "Are assigned in a odd-like sequence: 1, 3, 5, 7, 9...",
+    Distractor1: "Represent the contributions of the whole team",
+    Distractor2: "Estimate the ‘size’ of stories relative to each other",
+    Distractor3: "Do not equate directly to time the task might take",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:153]:
```suggestion
    Answer: "points are assigned in a odd-like sequence: 1, 3, 5, 7, 9...",
    Distractor1: "points represent the contributions of the whole team",
    Distractor2: "points estimate the ‘size’ of stories relative to each other",
    Distractor3: "points do not equate directly to time the task might take",
```

File: src/data/agile-quiz.ts [hunk 3]
```diff
@@ -144,6 +144,17 @@ const agileQuiz = [
...
+    Explanation:
+      "The estimate the amount of effort required to deliver a user story are assigned using Fibonacci sequence",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:155]:
```suggestion
      "Points estimate the amount of effort required to deliver a user story and are assigned using the Fibonacci sequence",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by PhilipeSousa** [comment]
I made the adjustments, thank you for your attention.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**